ServerInstall.md

```diff
@@ -0,0 +1,10 @@
+Server install
+==============
+
+To set up a functioning Chime server:
+
+1. Start up an EC2 instance
```

Comment:
Maybe add some details as to what type of instance? i.e. "Ubuntu Server 14.04 LTS (HVM), SSD Volume Type"